src/test/java/com/exasol/edmlgenerator/parquet/ParquetColumnToMappingDefinitionConverterTest.java

```diff
@@ -47,4 +110,96 @@ private void assertConvertsToToDecimalMapping(final Type type, final int scale,
                 () -> assertThat(toDecimalMapping.getDestinationName(), equalTo(name))//
         );
     }
+
+    @Test
+    void testConvertLogicalTypeTime() {
+        final Type logicalTimeType = Types.primitive(INT32, REQUIRED)
+                .as(LogicalTypeAnnotation.timeType(true, LogicalTypeAnnotation.TimeUnit.MILLIS)).named("myTime");
+        assertConvertsToToDecimalMapping(logicalTimeType, 0, 20, "MY_TIME");
+    }
+
+    @Test
+    void testConvertLogicalTypeTimestamp() {
+        final Type logicalTimestampType = Types.primitive(INT64, REQUIRED)
+                .as(LogicalTypeAnnotation.timestampType(true, LogicalTypeAnnotation.TimeUnit.MILLIS))
+                .named("myTimestamp");
+        final ToTimestampMapping toTimestampMapping = (ToTimestampMapping) new ParquetColumnToMappingDefinitionConverter()
+                .convert(logicalTimestampType);
+        assertAll(//
+                () -> assertThat(toTimestampMapping.getDestinationName(), equalTo("MY_TIMESTAMP")), //
+                () -> assertThat(toTimestampMapping.isUseTimestampWithLocalTimezoneType(), equalTo(true))//
+        );
+    }
+
+    @Test
+    void testConvertInt96ToTimestamp() {
+        final Type logicalTimestampType = Types.primitive(INT96, REQUIRED).named("myTimestamp");
+        final ToTimestampMapping toTimestampMapping = (ToTimestampMapping) new ParquetColumnToMappingDefinitionConverter()
+                .convert(logicalTimestampType);
+        assertAll(//
+                () -> assertThat(toTimestampMapping.getDestinationName(), equalTo("MY_TIMESTAMP")), //
+                () -> assertThat(toTimestampMapping.isUseTimestampWithLocalTimezoneType(), equalTo(true))//
+        );
+    }
+
+    @ParameterizedTest
+    @MethodSource("testConvertToStringCases")
+    void testConvertToString(final LogicalTypeAnnotation annotation) {
```

Comment:
Put testConvertToStringCases next to this method to make the code easier to read.

Reply:
Unfortunately the formatter moves static methods to the top. :-/